mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -122,7 +122,8 @@ def load_telemetry(self, trial_id: int) -> List[Tuple[datetime, str, Any]]:
             return [(row.ts, row.metric_id, row.metric_value)
                     for row in cur_telemetry.fetchall()]
 
-    def load(self, opt_target: Optional[str] = None) -> Tuple[List[dict], List[Optional[float]], List[Status]]:
+    def load(self, last_trial_id: int = -1,
```

Comment:
```suggestion
    def load(self,
             last_trial_id: int = -1,
```
nit: when breaking multiple params across lines, I find it easier to read when `self` or maybe `self, *,` are by themselves, and all others are on their own line.